apis/core/lib/auth.ts

```diff
@@ -12,15 +14,15 @@ declare module 'express-serve-static-core' {
   }
 }
 
-const createJwtHandler = (credentialsRequired: boolean) => jwt({
+const createJwtHandler = (credentialsRequired: boolean, jwksUri: string) => jwt({
   // Dynamically provide a signing key
   // based on the kid in the header and
   // the signing keys provided by the JWKS endpoint.
   secret: jwksRsa.expressJwtSecret({
     cache: true,
     rateLimit: true,
     jwksRequestsPerMinute: 5,
-    jwksUri: `${env.AUTH_ISSUER}/.well-known/jwks.json`,
```

Comment:
Reading this from `.well-known/openid-configuration` seeing how the JWKS endpoint has varying URL in oidc providers. So much or `.well-known` 🙄